src/main/resources/listLookup.json

```diff
@@ -557,5 +557,16 @@
         "restrictedProvenances": [],
         "defaultSensitivity": "",
         "isNonStrategic": true
+    },
+    "AST_DAILY_HEARING_LIST": {
+        "friendlyName": "Asylum Support Tribunal Daily Hearing List",
+        "welshFriendlyName": "Rhestr ddyddiol y Tribiwnlys Cefnogi Ceiswyr Lloches",
```

Comment:
I think the welsh will need updating here to account for the addition of Hearing into the name

Reply:
Similar comment with language files below